### PR TITLE
Fix debug instead of environment == dev

### PR DESCRIPTION
**Pull request by djgxp**

### CODE REVIEW

File: src/Services/ClientRegistry.php [hunk 1]
```diff
@@ -34,15 +34,15 @@ final class ClientRegistry
      * ClientRegistry constructor.
      *
      * @param EventDispatcherInterface $eventDispatcher
-     * @param string                   $environment
+     * @param bool                   $debug
      *
      * @internal param DataCollectorLoggerInterface $logger
      */
-    public function __construct(EventDispatcherInterface $eventDispatcher, string $environment)
+    public function __construct(EventDispatcherInterface $eventDispatcher, bool $debug)
```
Comment (Jean85) [inline src/Services/ClientRegistry.php]: This is the only BC. It's acceptable only in the 1.x release.

**Comment by Jean85** [comment]
Oh, it seems there's a duplicate, #84, but it has tests fixes... What should we do?

**Comment by djgxp** [comment]
Actually there is a fix for the branch 0.7 and the master one so that I did 2 pull requests. But definitely there are some tests to fix

**Comment by ilario-pierbattista** [comment]
Hi @djgxp thank you for your contribution.

I think we could proceed this way:

1. Merge #83 with master
2. Write a test to cover the case pointed out by @Jean85 (and eventually fix bugs)
3. release a new tag for the 1.x branch
4. decide what to do with #84

**Pull request merged**